### DISCUSSION TITLE
certifating student package

### DISCUSSION THREAD

**Discussion by leemj0522** [community/community#89997] [GitHub Education]
### Select Topic Area

Question

### Body

i upload transcript  for certifating student package so i press green button,but It keeps saying "processing information" and does not move on to the next process.how to solve this problem
![2024-01-20](https://github.com/community/community/assets/130457814/6478a0a5-46c6-4977-8542-6a9a8d1928c0)
![2024-01-20 (1)](https://github.com/community/community/assets/130457814/8fa59b37-d2b0-4a80-9cb7-086220a71e89)

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @leemj0522 , we're happy you're here! You are more likely to get a useful response if you are posting your questions in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by kaushal878** [comment]
Try to wait , or try other verification process
